.github/workflows/package-validation.yml

```diff
@@ -22,7 +22,7 @@ jobs:
       run: dotnet restore
 
     - name: Pack
-      run: dotnet pack --configuration Release --no-restore --property:EnablePackageValidation=true;ExposeExperimentalFeatures=false
+      run: dotnet pack --configuration Release --no-restore --property:EnablePackageValidation=true --property:ExposeExperimentalFeatures=false
```

Comment:
just a nit comment as I saw this. I think you are one of the few customers that use the --property syntax to pass properties to msbuild. Nearly everyone else uses `/p:` which is shorter.

Reply:
Thanks Viktor, sure no need to keep unnecessarily long :)